README.md

```diff
@@ -1,2 +1,28 @@
-# static-code-analysis-tool
-Tool for performing static code analysis for Ballerina projects.
+# Ballerina Static Code Analysis Tool
+
+## Overview
+
+Static Code Analysis (SCA) is an approach that uses tools to examine code without execution. They are used for identifying potential issues like bugs, vulnerabilities, and style violations. SCA improves software quality by early issue detection, ensuring better maintainability, and providing enhanced security. 
+
+The Ballerina scan tool provides a set of command line tools to statically analyze Ballerina files and report analysis issues.

```

Comment:
We can elaborate a bit more here saying this is a Ballerina tool. 

> provides a set of command line tools

isn't this rather "provides the command-line functionality to ..."

Also elaborate more on what exactly this repo contains. i.e., the fact that this consists of the core scan logic and extension points for analysis and reporting to platforms